snakemake_for_qc.md

```diff
@@ -0,0 +1,64 @@
+# Using snakemake to automate an RNA-seq workflow
+
+In our previous lesson, we use snakemake to automate the first two rules of our
+quality control/quality analysis. Given the things you learned during that 
+lesson, we will now automate the rest of the quality control  workflow through
+using snakemake. For this lesson, we would like you to work with others in 
+your room to make the snakefile. We have provided the basic structure for each 
+of the rules, and would like you to fill in the remaining necessary details for 
+the full workflow to run on all samples. 
+
+We've added in the rules we created from our previous lesson as well.
+
+Let's make sure we're in a good working directory and we have the necessary
+software installed.
```

Comment:
can this be optional? as I am guessing many will have all three tools already installed.

Reply:
yes, i include it there in case anyone started a new instance or has trouble logging on and needs to start a new one. want me to add language making it optional?

Reply:
yes, making it optional will be less confusing. Thanks